man/adoc/bpftrace.adoc

```diff
@@ -520,11 +520,11 @@ To improve the readability of big literals an underscore `_` can be used as fiel
 Integer suffixes as found in the C language are parsed by bpftrace to ensure compatibility with C headers/definitions but they're not used as size specifiers.
 `123UL`, `123U` and `123LL` all result in the same integer type with a value of `123`.
 
-Character literals can be defined by enclosing the character in single quotes e.g. `$c = 'c';`.
+Character literals are not supported at this time, and the corresponding ASCII code must be used instead.
```

Comment:
> corresponding ASCII code must be used instead.

Maybe I'm dense but I'm not sure what this means in this context. Do you have an example?

Reply:
`man ascii` - for example: `65` instead of `A`

Reply:
Ah ok, and this only works for `printf` as you need "%c".

Reply:
Yes, exactly, I was referring to that use-case (which is admittedly a niche one).

In my specific case, I'm outputting the TCP flags with something along the lines of:

```
#define CH_S 0x53   // 'S'
#define CH_A 0x41   // 'A'
...

printf("TCP flags: %c%c%c%c\n",
        $ip4h->protocol == PROTO_TCP && $tcph->syn ? CH_S : CH_DOT,
        $ip4h->protocol == PROTO_TCP && $tcph->ack ? CH_A : CH_DOT,
        $ip4h->protocol == PROTO_TCP && $tcph->fin ? CH_F : CH_DOT,
        $ip4h->protocol == PROTO_TCP && $tcph->rst ? CH_R : CH_DOT
)
```

I had initially used strings instead, but that turned out triggering the BPF stack limit exceeded error (given that each requires 64 bytes).

Reply:
I wouldn't be opposed to including a `printf` example just so users don't get confused (as I did) but I also won't block if someone else wants to merge.

Reply:
Last push added a very basic example.

Reply:
Thanks! Looks great.